prov/gni/include/gnix.h

```diff
@@ -391,16 +392,13 @@ struct gnix_fid_ep {
 	struct gnix_tag_storage tagged_unexp_recv_queue;
 	struct gnix_tag_storage tagged_posted_recv_queue;
 
-
-	fastlock_t recv_comp_lock;
-	struct slist pending_recv_comp_queue;
 	/* pointer to tag matching engine */
-	void *tag_matcher;
```

Comment:
I removed them at some point. They are completely unused.
-- Jim




On 4/14/16, 5:24 PM, "Sung-Eun Choi" <notifications@github.com> wrote:

>In 
>prov/gni/include/gnix.h
><https://github.com/ofi-cray/libfabric-cray/pull/776#discussion_r59802162>
>:
>>  	/* pointer to tag matching engine */
>> -	void *tag_matcher;
>Is the removal of these fields general clean up? Perhaps they were put in
>for future use? Not that I think they should be maintained, but if
>there's some useful background info, we should document it in the commit
>message.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly or
>view it on GitHub 
><https://github.com/ofi-cray/libfabric-cray/pull/776/files/e88688645fe4700
>d5e9ac9c2697f4a36b7bd0ab8#r59802162>
>
>